pkgs/by-name/pa/paratest/package.nix

```diff
@@ -3,19 +3,19 @@
 , lib
 }:
 
-(php.withExtensions ({ enabled, all }: enabled ++ [ all.pcov ])).buildComposerProject (finalAttrs: {
+(php.withExtensions ({ enabled, all }: enabled ++ [ all.pcov ])).buildComposerProject2 (finalAttrs: {
   pname = "paratest";
-  version = "7.4.3";
+  version = "7.6.0";
 
   src = fetchFromGitHub {
     owner = "paratestphp";
     repo = "paratest";
     rev = "v${finalAttrs.version}";
-    hash = "sha256-Shf/fsGhDmupFn/qERzXGg3ko7mBgUqYzafO/VPqmoU=";
+    hash = "sha256-p7m/MDHy+NOh+MnoIWy74Ipm/5hevp4x6Qwn4uPIEAM=";
   };
 
   composerLock = ./composer.lock;
```

Comment:
The `composer.lock` file hasn't changed?

Reply:
It changed but it's the same filename

Reply:
#facepalm ... I just noticed. Sorry for the noise.

Reply:
No worries, it happens, no need for excuses :)